Makefile.am

```diff
@@ -194,6 +196,41 @@ cc_oci_runtime_CFLAGS = \
 	$(LIBMNL_CFLAGS) \
 	$(UUID_CFLAGS)
 
+libexec_SCRIPTS = cc-proxy
+
+AM_V_GO    = $(am__v_GO_@AM_V@)
+am__v_GO_  = $(am__v_GO_@AM_DEFAULT_V@)
+am__v_GO_0 = @echo "  GOBUILD " $@;
+am__v_GO_1 =
+
+systemdservice_in_files =		\
+	proxy/cc-proxy.service.in	\
+	proxy/cc-proxy.socket.in	\
+	$(NULL)
```

Comment:
I guess this is to avoid the hassle of deciding whether to add a backslash each time the list is updated? If so, I'd still define `NULL` to be nothing to make it clearer (even better, with a comment explaining the intent ;-)